### PR TITLE
Removed WorkerResponse and ServerResponse

### LINKED ISSUE #95: Response classes should be unified
As discussed in the meting, I think we should have only a single Response class instead of Response, ServerResponse, and WorkerResponse.
Since we currently have a lot of PRs open this change should wait until we have a lower amount of PRs open.

### PR DESCRIPTION
**Pull request by nikolatzotchev**
fixes #95

### CODE REVIEW

File: src/ja/common/proxy/ssh.py [hunk 1]
```diff
@@ -16,21 +17,21 @@ class ISSHConnection(ABC):
     """
 
     @abstractmethod
-    def send_server_command(self, command: ServerCommand) -> ServerResponse:
+    def send_server_command(self, command: ServerCommand) -> Response:
```
Comment (JohannesGaessler) [inline src/ja/common/proxy/ssh.py]: Now that there is only one Response class, please unify send_server_command and send_worker_command.
I only created two separate methods due to the limitations imposed imposed by mypy.

File: src/ja/common/proxy/ssh.py [hunk 2]
```diff
@@ -17,21 +16,12 @@ class ISSHConnection(ABC):
     """
 
     @abstractmethod
-    def send_server_command(self, command: ServerCommand) -> ServerResponse:
+    def send_command(self, command: Command) -> Response:
         """!
         Sends a ServerCommand to the Remote on the host. Automatically attaches the username specified in ssh_config to
```
Comment (ammen99) [inline src/ja/common/proxy/ssh.py]: Sends a Command*

File: src/ja/common/proxy/ssh.py [hunk 3]
```diff
@@ -17,21 +16,12 @@ class ISSHConnection(ABC):
     """
 
     @abstractmethod
-    def send_server_command(self, command: ServerCommand) -> ServerResponse:
+    def send_command(self, command: Command) -> Response:
         """!
-        Sends a ServerCommand to the Remote on the host. Automatically attaches the username specified in ssh_config to
+        Sends a Command to the Remote on the host. Automatically attaches the username specified in ssh_config to
         the ServerCommand.
```
Comment (ammen99) [inline src/ja/common/proxy/ssh.py]: still a ServerCommand here :P

**Comment by nikolatzotchev** [comment]
will do

**Comment by ammen99** [comment]
needs a rebase

**Pull request merged**